### PR TITLE
Sortby component documentation

### PR DESCRIPTION
**Pull request by konarshankar07**
## Purpose of this pull request

This pull request (PR) will add the documentation for sortBy component which introduced in Magento 2.4 

### Related PR

https://github.com/magento/magento2/pull/26736

Added a new topic for the [Sortby](https://devdocs.magento.com/guides/v2.4//ui_comp_guide/components/ui-sortby.html) UI component.

### CODE REVIEW

**Comment by konarshankar07** [comment]
Hello @sivaschenko ...
Please review
Thanks

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by jeff-matthews** [comment]
Thanks for the PR @konarshankar07! 

I'm adding @sivaschenko for technical review and @hguthrie for editorial review.

File: src/guides/v2.4/ui_comp_guide/components/ui-sortby.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+group: ui-components-guide
+title: Sortby component
+contributor_name: Shankar Konar
+contributor_link: https://github.com/konarshankar07
+---
+
+The Sortby component allows to sort the column in ascending or descending order.
```
Comment (hguthrie) [inline src/guides/v2.4/ui_comp_guide/components/ui-sortby.md:8]:
```suggestion
The Sortby component allows you to sort the column in ascending or descending order.
```

File: src/guides/v2.4/ui_comp_guide/components/ui-sortby.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+## Configuration options
+
+|      Option              |   Description                                                         |      Type      |  Default Value |
+|--------------------------|-----------------------------------------------------------------------|----------------| ---------------|
+| template                 | Path to the component’s <code>.html</code> template.                  | String         | ui/grid/sortBy |
```
Comment (hguthrie) [inline src/guides/v2.4/ui_comp_guide/components/ui-sortby.md:14]: Please use the inline code format for options and default examples.

```suggestion
| `template`                 | Path to the component `.html` template.                  | String         | `ui/grid/sortBy` |
```

File: src/guides/v2.4/ui_comp_guide/components/ui-sortby.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+| options                  | List of options available for sorting.                                | Array          | -              |
+| applied                  | Currently applied sorting.                                            | Object         | -              |
+| sorting                  | Column's sorting order. Can be ascending (`asc`), descending (`desc`).| String         | asc            |
```
Comment (hguthrie) [inline src/guides/v2.4/ui_comp_guide/components/ui-sortby.md:17]:
```suggestion
| `sorting`                  | Specify ascending (`asc`) or descending (`desc`) sorting order for the column.| String         | `asc`            |
```

File: src/guides/v2.4/ui_comp_guide/components/ui-sortby.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+---
+
+The Sortby component allows you to sort the column in ascending or descending order.
+
+## Configuration options
+
+|      Option                |   Description                                                                 |      Type      |  Default Value   |
+|----------------------------|-------------------------------------------------------------------------------|----------------| -----------------|
+| `template`                 | Path to the component .html template.                                         | String         | `ui/grid/sortBy` |
+| `options`                  | List of options available for sorting.                                        | Array          | `-`              |
```
Comment (hguthrie) [inline src/guides/v2.4/ui_comp_guide/components/ui-sortby.md:15]: If this is not an actual default, you don't need the inline code formatting for the `-`.

```suggestion
| `options`                  | List of options available for sorting.                                        | Array          | `-`              |
```

Comment (rogyar) [inline src/guides/v2.4/ui_comp_guide/components/ui-sortby.md:15]: Please, use the exact default value

```suggestion
| `options`                  | List of options available for sorting.                                        | Array          | `[]`              |
```

File: src/guides/v2.4/ui_comp_guide/components/ui-sortby.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+| `applied`                  | Currently applied sorting.                                                    | Object         | `-`              |
```
Comment (rogyar) [inline src/guides/v2.4/ui_comp_guide/components/ui-sortby.md:16]: Please, use the exact default value
```suggestion
| `applied`                  | Currently applied sorting.                                                    | Object         | `{}`              |
```

**Comment by jeff-matthews** [comment]
@rogyar, what do you think?

File: src/guides/v2.4/ui_comp_guide/components/ui-sortby.md (hunk 4 above)
Comment (jeff-matthews) [reply src/guides/v2.4/ui_comp_guide/components/ui-sortby.md]: Change has been applied

File: src/guides/v2.4/ui_comp_guide/components/ui-sortby.md (hunk 5 above)
Comment (jeff-matthews) [reply src/guides/v2.4/ui_comp_guide/components/ui-sortby.md]: Change has been applied

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**